gcp/k8s/main.tf

```diff
@@ -18,15 +18,19 @@ resource "google_container_cluster" "mockten_k8s_cluster" {
 
   private_cluster_config {
     enable_private_nodes = true
-    enable_private_endpoint = true
+    enable_private_endpoint = false
     master_ipv4_cidr_block = var.k8s_master_cidr
 
     master_global_access_config {
-    enabled = false
+      enabled = false
     }
   }
 
   master_authorized_networks_config {
+    cidr_blocks {
+      cidr_block   = "192.0.2.127/32"
```

Comment:
This is test IP-address. (https://livra.geolocation.co.jp/iplearning/227/)
It often change our laptop global IP address, so we need to add our laptop IP address to autorized-network by using GCP UI.